activesupport/test/subscriber_test.rb

```diff
@@ -16,26 +14,17 @@ def open_party(event)
     events << event
   end
 
+  def another_open_party(event)
+    events << event
+  end
+
   private
     def private_party(event)
       events << event
     end
 end
 
-class TestSubscriber2 < ActiveSupport::Subscriber
-  attach_to :doodle
-  detach_from :doodle
-
-  cattr_reader :events
-
-  def self.clear
-    @@events = []
-  end
-
-  def open_party(event)
-    events << event
-  end
-end
+class PartySubscriber < TestSubscriber; end
```

Comment:
I think it is missing a test for the case where `TestSubscriber` was already attached but we want to attach a subclass of it adding new behavior without triggering the original behavior.

Reply:
Yes, good call!